src/OpenTelemetry/Metrics/MetricPoint.cs

```diff
@@ -258,9 +375,42 @@ internal void Update(double number)
 
                         lock (this.histogramBuckets.LockObject)
                         {
-                            this.primaryValue.CurrentAsLong++;
-                            this.secondaryValue.CurrentAsDouble += number;
-                            this.histogramBuckets.BucketCounts[i]++;
+                            this.runningValue.AsLong++;
+                            this.histogramBuckets.RunningSum += number;
+                            this.histogramBuckets.RunningBucketCounts[i]++;
+                        }
+
+                        break;
+                    }
+
+                case AggregationType.HistogramWithMinMax:
```

Comment:
This case shares a lot of code with `case AggregationType.Histogram`
Should we try to dedupe it?